mlflow/server/auth/db/migrations/versions/8606fa83a998_initial_migration.py

```diff
@@ -0,0 +1,64 @@
+"""initial_migration
+
+Revision ID: 8606fa83a998
+Revises: 
+Create Date: 2023-07-07 23:30:50.921970
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "8606fa83a998"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "users",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("username", sa.String(length=255), nullable=True),
+        sa.Column("password_hash", sa.String(length=255), nullable=True),
+        sa.Column("is_admin", sa.Boolean(), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("username"),
+    )
+    op.create_table(
+        "experiment_permissions",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("experiment_id", sa.String(length=255), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("permission", sa.String(length=255), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["user_id"],
+            ["users.id"],
+        ),
```

Comment:
The foreign key constraint alias can be supplied like this:

`sa.ForeignKeyConstraint(["user_id"],["users.id"], name="fk_user_id")`